src/utilities/dom.utilities.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Set a specific attribute on a group of elements
+ * @param {NodeList} elms collection of nodes, usually from document.querySelectorAll
+ * @param {string} attributeName name of the attribute
+ * @param {*} attributeValue attribute value
+ */
+const SetAttribute = (elms, attributeName, attributeValue) => {
+  elms.forEach(elm => {
+    elm.setAttribute(attributeName, attributeValue);
+  });
+};
+
```

Comment:
Might be good to expose the RemoveAttribute as well for consitency.

Reply:
i don't disagree, but I didn't have a need for it yet (i dont' think).